CWMS/utils.py

```diff
@@ -14,65 +15,49 @@ def queryCDA(self, endpoint, payload, headerList, return_type, dict_key):
         query parameters passed to ``requests.get``
     headerList: dict
         headers
-    return_type : str
-        output type to return values as. 1. 'df' will return a pandas dataframe. 2. 'dict' will return a json decoded dictionay. 3. all other values will return Responce object from request package.
-    dict_key : str
-        key needed to grab correct values from json decoded dictionary.
-        
 
     Returns
     -------
     string: query response
         The response from the API query ``requests.get`` function call.
     """
 
+    response = self.get_session().get(endpoint, params=payload, headers=headerList)
 
-    response = self.get_session().get(endpoint, params=payload, headers=headerList, verify=False)
+    if response.status_code > 200:
```

Comment:
How about `response.raise_for_status()`?

Reply:
for now I need the responce.text.  That way I can check CloudWatch and see what the errors are if we get them.  The raise_for_status might be usefull long term but it doesn't provide the '{"message":"Not found.","incidentIdentifier":"4723449190288442384","details":{}}' information from CDA for debugging.

Reply:
Ok, we'll probably want to create different exceptions for 404 anyways so that users can choose to ignore missing data. I like that the HTTPError includes the response object itself so that downstream users can pull metadata from the response as needed. 

I also would rather throw an exception subclass so that calling code can handle different errors. I think throwing the HTTPException with the modified error message would work fine.

Reply:
we can adjust the error handling at a later date.  to keep things moving I will leave at the moment.

Reply:
yup!